### DISCUSSION TITLE
inquiry and presentation

### DISCUSSION THREAD

**Discussion by angeangoh** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#7] [Outreachy 2022] [answered]
Hello @BatoolMM and the community. I am Ange Angoh an outreachy intern, I am Cameroonian and really happy to be there. I wish to contribute and hope I will be an amazing journey. Anyway I am currently unable to fill the form to join the community channel. I received this message `This form has exceeded its allocated quota.` please for your help.

**Comment by BatoolMM** [comment, marked as answer]
Hi @angemaggy, 
Apologies for the broken link, is [this one](https://form.jotform.com/221813832742353) working for you?

**Reply by angeangoh** [reply]
yes thank you
